OpenRA.Mods.Common/Traits/World/MusicPlaylist.cs

```diff
@@ -54,6 +55,10 @@ public MusicPlaylist(World world, MusicPlaylistInfo info)
 			this.info = info;
 			this.world = world;
 
+			// Reload music with map assets mounted
+			foreach (var entry in world.Map.Rules.Music)
+				entry.Value.Reload();
```

Comment:
Calling `Reload` for each is needed since any could be overridden or added by assets in the map. I am looking into somewhere else.